runtime/src/snapshot_bank_utils.rs

```diff
@@ -1962,11 +1963,18 @@ mod tests {
         let full_snapshot_archives_dir = tempfile::TempDir::new().unwrap();
         let incremental_snapshot_archives_dir = tempfile::TempDir::new().unwrap();
 
-        let genesis_config_info = genesis_utils::create_genesis_config_with_leader(
+        let mut genesis_config_info = genesis_utils::create_genesis_config_with_leader(
             1_000_000 * LAMPORTS_PER_SOL,
             &Pubkey::new_unique(),
             100 * LAMPORTS_PER_SOL,
         );
+        // When the accounts lt hash feature is enabled, the incremental accounts hash is bypassed.
```

Comment:
so when this feature is activated, we don't need to compute incremental hash for incremental snapshot. We will use the checksum of bank' lt hash directly? 
Has this change been implemented already? Looks like I can't find it in this PR or in master.

Reply:
I thought this feature will only affect EAH. For snapshots, we will still using old merkle tree hash. 
If so, why incremental hash is bypassed with this feature?